apps/client/src/app/components/admin-market-data/asset-profile-dialog/asset-profile-dialog.component.ts

```diff
@@ -223,6 +250,38 @@ export class AssetProfileDialog implements OnDestroy, OnInit {
       });
   }
 
+  private isNewSymbolValid(control: AbstractControl): ValidationErrors {
+    const currentAssetProfileIdentifier: AssetProfileIdentifier | undefined =
+      control.get('symbol').value;
+
+    if (
+      currentAssetProfileIdentifier?.dataSource === this.data?.dataSource &&
+      currentAssetProfileIdentifier?.symbol === this.data?.symbol
+    ) {
+      return {
+        equalsPreviousSymbol: true
+      };
+    }
+  }
+
+  public get isSymbolEditable() {
+    return !this.assetProfileForm.dirty;
+  }
+
+  public get isSymbolEditButtonInvisible() {
```

Comment:
`isAssetProfileIdentifierEditButtonVisible`

I would use _visible_ as the default value.